### PR TITLE
686 Remove UK references and styles

### PR DESCRIPTION
**Pull request by jonathanbobel**

### CODE REVIEW

**Comment by jonathanbobel** [comment]
I still have more to do to strip out all mentions of govuk styles, but want to spin up another PR to do so so it's not as much to sift through

File: app/templates/views/user-profile/manage-security-key.html
Comment (stvnrlly) [inline app/templates/views/user-profile/manage-security-key.html:1]: I believe this was removed recently and can stay gone.

File: app/templates/views/user-profile/security-keys.html
Comment (stvnrlly) [inline app/templates/views/user-profile/security-keys.html:1]: I believe this was removed recently and can stay gone.

File: app/templates/views/two-factor-webauthn.html
Comment (stvnrlly) [inline app/templates/views/two-factor-webauthn.html:1]: I believe this was removed recently and can stay gone.

File: app/templates/views/platform-admin/_base_template.html [hunk 1]
```diff
@@ -2,41 +2,49 @@
...
-            ('Trial mode services', url_for('main.trial_services')),
-            ('Organizations', url_for('main.organizations')),
-            ('Providers', url_for('main.view_providers')),
-            ('Reports', url_for('main.platform_admin_reports')),
-            ('Email branding', url_for('main.email_branding')),
-            ('Inbound SMS numbers', url_for('main.inbound_sms_admin')),
-            ('Find services by name', url_for('main.find_services_by_name')),
-            ('Find users by email', url_for('main.find_users_by_email')),
-            ('Email complaints', url_for('main.platform_admin_list_complaints')),
-            ('Clear cache', url_for('main.clear_cache')),
```
Comment (stvnrlly) [inline app/templates/views/platform-admin/_base_template.html:27]: Why remove the `url_for`? It seems like that might make things more brittle.

File: tests/javascripts/templateFolderForm.test.js [hunk 2]
```diff
@@ -624,7 +624,7 @@ describe('TemplateFolderForm', () => {
 
       test("the content of the counter should reflect the selection", () => {
 
-        expect(visibleCounterText).toEqual('1 template, 1 folder selected');
+        expect(visibleCounterText).toEqual('selected');
```
Comment (stvnrlly) [inline tests/javascripts/templateFolderForm.test.js:627]: Is this to be expected? This implies that some of the content disappeared (unless there's another change in the template)

Comment (jonathanbobel) [reply tests/javascripts/templateFolderForm.test.js:627]: I can revisit this, but it was breaking the test even though on the actual page it is showing the '1 template, 1 folder' wording. Might reach out for a second pair of eyes, as I couldn't figure out why this wasn't working earlier.

File: app/templates/views/platform-admin/_base_template.html [hunk 3]
```diff
@@ -2,41 +2,49 @@
...
+              ('Find services by name', ('main.find_services_by_name')),
+              ('Find users by email', ('main.find_users_by_email')),
+              ('Email complaints', ('main.platform_admin_list_complaints')),
+              ('Clear cache', ('main.clear_cache')),
+            ] %}
+            <li class="usa-sidenav__item">
+              <a class="{% if url == request.endpoint %} usa-current {% endif %}" href="{{ url_for(url) }}">
+                <span class="usa-sidenav__label">{{ link_text }}</span>
               </a>
             </li>
```
Comment (jonathanbobel) [inline app/templates/views/platform-admin/_base_template.html:40]: @stvnrlly I moved the _url_for_ to the actual link so the side navigation styling works as expected. I don't think it affects its usage.

File: app/templates/views/user-profile/manage-security-key.html
Comment (jonathanbobel) [reply app/templates/views/user-profile/manage-security-key.html:1]: Are you referring to the whole file?

File: app/templates/views/platform-admin/_base_template.html (hunk 1 above)
Comment (jonathanbobel) [reply app/templates/views/platform-admin/_base_template.html:27]: See comment below

File: app/templates/views/user-profile/manage-security-key.html
Comment (stvnrlly) [reply app/templates/views/user-profile/manage-security-key.html:1]: Yeah, Carlo took these out in https://github.com/GSA/notifications-admin/pull/697, if I'm not confusing myself.

File: app/templates/views/platform-admin/_base_template.html (hunk 3 above)
Comment (stvnrlly) [reply app/templates/views/platform-admin/_base_template.html:40]: Ah, perfect! That's my oversight, thanks.

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**